### ISSUE TITLE
Bad value substitution in ini_file Module with percent placeholder values

### ISSUE THREAD

**Issue by viktorbreitenbuecher** [ansible/ansible-modules-core#627]
##### Issue Type:

Bug
##### Component Name:

ini_file module
##### Ansible Version:

1.7.2
1.8.2
##### Environment:

Mac OSX 10.10.1 Yosemite
##### Summary:

I need special values in an ini-file to configure my supervisor-daemon. For example a value looks like this:
process_name=%(program_name)s
##### Steps To Reproduce:

To reporduce this issue run following playbook twice.

```

---
- hosts: all
  tasks:
    - ini_file: dest="/tmp/tmp.ini" section="program:update" option="process_name" value="%(program_name)s"
```
##### Expected Results:

After first run everything OK, after second run I get an error and nothing happens.
##### Actual Results:

```
ConfigParser.InterpolationMissingOptionError: Bad value substitution:
    section: [program:update]
    option : process_name
    key    : program_name
    rawval : %(program_name)s
```

**Comment by ansibot** [comment]
Can You Help Us Out?
====================

Thanks for filing a ticket! I am the friendly GitHub Ansibot. 

It looks like you might not have filled out the issue description based on our standard issue template.  You might not have known about that, and that's ok too, we'll tell you how to do it.

We have a standard template because Ansible is a really busy project and it helps to have some standard information in each ticket, and GitHub doesn't yet provide a standard facility to do this like some other bug trackers.  We hope you understand as this is really valuable to us!. 

Solving this is simple: please copy the contents of this [template](https://raw.githubusercontent.com/ansible/ansible/devel/ISSUE_TEMPLATE.md) and **paste it into the description** of your ticket.  That's it!

If You Had A Question To Ask Instead
====================================

If you happened to have a "how do I do this in Ansible" type of question, that's probably more of a user-list question than a bug report, and you should probably ask this question on the project mailing list instead.  

However, if you think you have a bug, the report is the way to go!  We definitely want all the bugs filed :)  Just trying to help!

About Priority Tags
===================

Since you're here, we'll also share some useful information at this time.

In general tickets will be assigned a priority between P1 (highest) and P5, and then worked in priority order. We may also have some follow up questions along the way, so keeping up with follow up comments via GitHub notifications is a good idea.

Due to large interest in Ansible, humans may not comment on your ticket immediately.  

Mailing Lists
=============

If you have concerns or questions, you're welcome to stop by the ansible-project or ansible-development mailing lists, as appropriate.  Here are the links:

  * https://groups.google.com/forum/#!forum/ansible-project - for discussion of bugs and how-to type questions
  * https://groups.google.com/forum/#!forum/ansible-devel - for discussion on how to implement a code change, or feature brainstorming among developers

Thanks again for the interest in Ansible!

**Comment by magnetik** [comment]
Confirming it. I'm trying to set the control_path of ansible.cfg (with ansible), and I'm facing the same issue.

**Comment by ansibot** [comment]
@jpmens, @noseka1, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by noseka1** [comment]
Could you try with Ansible >= 2.0? It might have been fixed there.

**Comment by vundb** [comment]
Works fine with ansible 2.1.1.0.

**Comment by ansibot** [comment]
@jpmens, @noseka1, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by noseka1** [comment]
wontfix

User can upgrade to Ansible >= 2.0 where this is not an issue.
